src/config/babelrc.js

```diff
@@ -8,9 +10,21 @@ const isWebpack = parseEnv('BUILD_WEBPACK', false)
 const treeshake = parseEnv('BUILD_TREESHAKE', isRollup || isWebpack)
 const alias = parseEnv('BUILD_ALIAS', isPreact ? {react: 'preact'} : null)
 
+/**
+ * use the strategy declared by browserslist to load browsers configuration.
+ * fallback to the default if don't found custom configuration
+ * @see https://github.com/browserslist/browserslist/blob/master/node.js#L139
+ */
+const browsersConfig = browserslist.loadConfig({path: './'}) || [
```

Comment:
The path should probably be `rootDir` from `utils`. I think... Right? Have you tried this in your project?